platform-server-kernel/src/main/java/uno/cod/platform/server/core/dto/participation/ParticipationShowDto.java

```diff
@@ -0,0 +1,60 @@
+package uno.cod.platform.server.core.dto.participation;
+
+import uno.cod.platform.server.core.domain.Participation;
+import uno.cod.platform.server.core.dto.challenge.ChallengeShortShowDto;
+import uno.cod.platform.server.core.dto.team.TeamShowDto;
+import uno.cod.platform.server.core.dto.user.UserShowDto;
+
+import java.util.Date;
+
+public class ParticipationShowDto {
+    private UserShowDto user;
+    private TeamShowDto team;
```

Comment:
So you are suggesting to defer a change that is straightforward but might add a DTO that is a bit too large until we implemented some IndexDB magic on the client side? WTF?